### ISSUE TITLE
Build Picture too big.

### ISSUE THREAD

**Issue by kul111** [buildsgg/buildsgg#2]
https://builds.gg/builds/project-khronos-3

cover picture to big - need to scroll even on my big screen - should we set a size limit in regards to how big pictures are being displayed.

**Comment by bashgeek** [comment]
Yeah, that's certainly an issue with pictures in portrait orientation. Will figure something out
